packages/babel-preset-gatsby/src/index.js

```diff
@@ -114,7 +119,7 @@ export default function preset(_, options = {}) {
           // absoluteRuntime: absoluteRuntimePath,
         },
       ],
-      [
+      isBrowser && [
```

Comment:
Will `class-transform` be provided when `isBrowser` is `true`? If not, Babel will still throw on `super(...args)`. Before 7.12.11, babel outputs untranspiled `super(...args)` without any warnings.

Reply:
It will work unless people have a custom browserslist where spread is not needed 🤔. Good call

Reply:
Consider switch to [`assumptions`](https://github.com/babel/babel/pull/12470) after Babel 7.13 is released. Ideally what we need is if you enable `spread-transform` because of your specified targets, do not assume "iterable is array", which what the `loose` mode of `spread-transform` did. Here we are unconditionally transpiling `...` even if the target already supports it.

Reply:
Awesome! Didn't know about assumptions yet. That will fix it. In the next major I'll add loose as a parameter so people can configure preset-env so we don't have to add a separate module for spread.